packages/wdio-junit-reporter/tests/__snapshots__/reporter.test.ts.snap

```diff
@@ -26,6 +26,8 @@ exports[`wdio-junit-reporter generates xml output correctly when having classNam
 
 exports[`wdio-junit-reporter generates xml output correctly when having classNameFormat override with mocha 1`] = `"<?xmlversion=\\"1.0\\"encoding=\\"UTF-8\\"?><testsuites><testsuitename=\\"Myawesomefeature\\"timestamp=\\"2018-04-17T09:10:10.255Z\\"time=\\"0.745\\"tests=\\"1\\"failures=\\"1\\"errors=\\"1\\"skipped=\\"0\\"><properties><propertyname=\\"specId\\"value=\\"0\\"/><propertyname=\\"suiteName\\"value=\\"Myawesomefeature\\"/><propertyname=\\"capabilities\\"value=\\"chrome.65_0_3325_181.macosx\\"/><propertyname=\\"file\\"value=\\"/path/to/project/test/specs/sync.spec.js\\"/></properties><testcaseclassname=\\"foo-chrome.65_0_3325_181.macosx-Myawesomefeature\\"name=\\"shouldcandosomething\\"time=\\"0.727\\"><failure/><errormessage=\\"Received:&quot;5&quot;\\"/><system-err><![CDATA[Received:\\"5\\"atContext.<anonymous>(/some/file.js:10:15)]]></system-err></testcase></testsuite></testsuites>"`;
 
+exports[`wdio-junit-reporter generates xml output correctly when having nested array of suites 1`] = `"<?xmlversion=\\"1.0\\"encoding=\\"UTF-8\\"?><testsuites/>"`;
+
```

Comment:
It seems that this is different from the v8 PR: https://github.com/webdriverio/webdriverio/pull/10977/files#diff-5b34bd882ef4247fe65465b8177c1d3411c1d264c927d204897f6b78f0b4a8f2R29

Reply:
Oops sorry this happens because of a new test snapshot written automatically.